src/pyspextool/telluric/load_spectra.py

```diff
@@ -277,13 +294,23 @@ def load_data():
         message = 'The standard lacks an order the object has.'
         raise pySpextoolError(message)
 
+    # Determine the angular separation between the object and standard
+
+    obj_long = ten(object_hdrinfo['RA'][0])*15*np.pi/180.
+    obj_lat = ten(object_hdrinfo['DEC'][0])*np.pi/180.
+    std_long = ten(standard_hdrinfo['RA'][0])*15*np.pi/180.
+    std_lat = ten(standard_hdrinfo['DEC'][0])*np.pi/180.   
```

Comment:
make SkyCoord objects and then you don't even need to convert to anything. I think `angular_separation` will figure it out.

Reply:
I have tried to avoid using other packages as much as possible. Things change, and this way if I write it, then I know what it does. Obviously that doesn't work for everything, but for this kind of stuff I want to do it this way.

Reply:
You're already using astropy.units and coordinates. This would not be adding an additional dependency. Right now, having this conversion done by hand is too easy to mess up.  Astropy.coordinates should be used for coordinate conversions.  Astropy.coordinates is used by observatories and missions.  I'm gonna hold my ground on this one.

Reply:
It is going to take me a bunch of time to make sure all of this works and I would prefer to move on to the H lines because that currently isn't even working.  Can we put this as an issue and I can come back to it after?  This is all because I am trying to compute the angular separation between the standard and object and is only used for QA.